roles/openshift_storage_glusterfs/tasks/main.yml

```diff
@@ -0,0 +1,192 @@
+---
+- name: Create temp directory for doing work in
+  command: mktemp -d /tmp/openshift-glusterfs-ansible-XXXXXX
+  register: mktemp
+  changed_when: False
+  check_mode: no
+
+- name: Copy the admin client config(s)
```

Comment:
Understood. Please add a comment to note that so no one tries to port it until missing items are added.